packages/orchestrator-ui-components/src/components/WfoSubscription/utils/utils.ts

```diff
@@ -107,13 +107,13 @@ export const getLastUncompletedProcess = (
             return dateB.getTime() - dateA.getTime();
         });
 
-    return uncompletedProcesses.length > 0
+    return (uncompletedProcesses && uncompletedProcesses.length) > 0
         ? uncompletedProcesses[0]
         : undefined;
 };
 
 export const getLatestTaskDate = (processes: SubscriptionDetailProcess[]) => {
-    if (processes.length === 0) {
+    if (!processes || processes.length === 0) {
```

Comment:
process cannot be null, see the type of the parameter, counts for all in this PR